src/generate.tsx

```diff
@@ -374,8 +374,7 @@ export default function generateSelector<
     const [innerValue, setInnerValue] = React.useState<ValueType>(
       value || defaultValue,
     );
-    const baseValue =
-      value !== undefined && value !== null ? value : innerValue;
+    const baseValue = 'value' in props ? value : innerValue;
```

Comment:
ref: ant-design/ant-design#20979 #443 You can not fix it in this way!

Reply:
maybe my understanding of this code was wrong...

Reply:
I think we should delete `&& value !== null` to fix it.

Reply:
@zombieJ